active_record_host_pool.gemspec

```
@@ -24,7 +24,8 @@ Gem::Specification.new do |s|
   s.test_files = ["test/database.yml", "test/helper.rb", "test/schema.rb", "test/test_arhp.rb"]
   s.license = "MIT"
 
-  s.add_runtime_dependency("activerecord", ">= 4.2.0", "< 6.1")
+  s.required_ruby_version = Gem::Requirement.new(">= 2.6.0")
```

Why use `Gem::Requirement`, was the string not sufficient? e.g.
```suggestion
  s.required_ruby_version = '>= 2.6.0'
```

---

habits 😅